docs/guides/manage-translations.rst

```diff
@@ -197,21 +197,23 @@ Once you have done changes in your documentation, you may want to make these add
 
 #. Create the ``.pot`` files:
 
-   .. code-block:: console
+   .. prompt:: bash $
 
-      $ sphinx-build -b gettext . _build/gettext
+      sphinx-build -b gettext . _build/gettext
 
 
-.. For the manual workflow, we need to run this command
+   For the manual workflow, we need to run this command
```

Comment:
We need the `..` to mark it as a comment as it was before.

Reply:
That will give rise to this condition ([comment](https://github.com/rtfd/readthedocs.org/pull/4990#issuecomment-446541187))

Should we hardcode the serial numbers?

Reply:
Is there no way to make it work properly? Did you try different new lines spacing?

It should work in some way because the comment should be ignored, I suppose.

We don't need to show that comment to the end user (I think I added), since it's something internal.

Reply:
The changes that i have pushed are seems to be working fine.
I tested them here: http://livesphinx.herokuapp.com/